modules/swagger-codegen/src/main/resources/dart/api_client.mustache

```diff
@@ -103,7 +93,7 @@ class ApiClient {
     } else if (obj is String) {
```

Comment:
This _if_, as a whole, should be unnecessary. JSON.convert(.) should be able to cope with all those types out of the box.